coupons.js

```diff
@@ -59,6 +59,10 @@ function handleSingleItemPage(priceboxdiv) {
 function displayCoupons() {
     var priceboxdivs = document.body.querySelectorAll('.price-box');
 
+    if (!priceboxdivs.length) {
+        priceboxdivs = document.body.querySelectorAll("div[class^='price__info']");
```

Comment:
because of the css class name hashing I mentioned before, this one work... I'm working on something like - 

```
querySelectorAll('div').each(function(i, t) {
   if (i.className.match(/price__info$/)) {
      priceboxdivs[] =  i;
...
```